aiida/orm/nodes/data/code/installed.py

```diff
@@ -69,9 +69,14 @@ def validate_filepath_executable(self):
         is intentionally not called in ``_validate`` as to allow the creation of ``Code`` instances whose computers can
         not yet be connected to and as to not require the overhead of opening transports in storing a new code.
 
+        .. note:: If the ``filepath_executable`` is not an absolute path, the check is skipped.
+
         :raises `~aiida.common.exceptions.ValidationError`: if no transport could be opened or if the defined executable
             does not exist on the remote computer.
         """
+        if not self.filepath_executable.is_absolute():
+            return
```

Comment:
just checking: what effect does this have in `verdi computer test` (or wherever this validation is used)?

will the check be shown as skipped or as passed?

Reply:
This check is only run by `verdi code test`.

Reply:
@sphuber I would note that ideally what you want to do, if the path is not absolute, is load the same environment that you would have, then check that the executable is on the `$PATH`.
Perhaps there could be an option in the code test, to run the prepend text (from both computer and code) then look for the executable
this would be a more "complete" test of whether the code might have issues running